vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/ItemClickEvent.java

```diff
@@ -101,4 +158,13 @@ public T getItem() {
         return item;
     }
 
+    /**
+     * Gets the column that was clicked.
+     * @return The column that was clicked. It should not be {@code null}, unless somehow the information
+     * about the column has not been passed from the client-side component.
```

Comment:
Formatting:
- empty line before `@return`
- no complete sentences in `@return`

>It should not be {@code null}, unless somehow....


^This is pretty un-conventional documentation. It's basically saying that it shouldn't be null, but there might be a bug which we're not aware of. The same applies to all of the APIs we have. :D
Change to e.g. "the column that was clicked, not {@code null}"